static/src/javascripts/projects/commercial/commercial-metrics.ts

```diff
@@ -35,7 +39,7 @@ const init = (): Promise<void> => {
  * It is safe to call this method repeatedly, as per [the `EventListener` docs](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#multiple_identical_event_listeners)
  */
 const captureCommercialMetrics = (): void => {
-	document.addEventListener('visibilitychange', sendMetrics);
+	bypassCommercialMetricsSampling();
 };
```

Comment:
I would drop this method entirely and simply replace it everywhere with the new `bypassCommercialMetricsSampling()`.